src/transformers/models/mamba2/__init__.py

```diff
@@ -33,6 +33,7 @@
 else:
     _import_structure["modeling_mamba2"] = [
         "Mamba2ForCausalLM",
+        "Mamba2ForSequenceClassification"
```

Comment:
"Mamba2ForSequenceClassification" sould have comma afterwards so
`"Mamba2ForSequenceClassification",`

Reply:
I got no idea how my IDE missed this, fixed.

Reply:
No worries. Nasty bug tbh and it wasn't shown in mine either :cry: